Library/Formula/libswiften.rb

```diff
@@ -1,24 +1,13 @@
 class Libswiften < Formula
   desc "C++ library for implementing XMPP applications"
   homepage "https://swift.im/swiften"
-  revision 2
 
   stable do
-    url "https://swift.im/downloads/releases/swift-2.0/swift-2.0.tar.gz"
-    sha256 "cbcdbe527dc4d112a38f3cdec5f1051d3beba0b97f8082f90debd04a5b45a41f"
+    url "git://swift.im/swift", :tag => "swift-3.0", :revision => "14ddf8b470f5a3420b5f2c96daea33c2513cac6e"
```

Comment:
For some reason, the tarball contains something different than the specified commit. [Contacted upstream](https://www.facebook.com/Swift.im/posts/919440988176623?comment_id=921744161279639) / @tfar about it.